frontend/src/components/Registration/LoginIndex.tsx

```diff
@@ -22,9 +25,24 @@ const LoginIndex = () => {
   const { backgroundColor, borderColor } = useThemeColor();
 
   const handleOnSubmit = (values: ILoginForm) => {
-    console.log(values);
+    handleLogin.mutate(values);
   };
 
+  const handleLogin = useMutation(
+    async (values: ILoginForm) => {
+      const res = await axios.post(`${baseURL}/auth/login`, values);
+      return res.data;
+    },
+    {
+      onSuccess: (res) => {
+        console.log("Success ", res);
+      },
+      onError: () => {
+        console.log("Error");
```

Comment:
don't commit console logs!

We can add comment that we will do error handling or success handling later